frontend/server/src/test/resources/snapshots/snapshot9.cfg

```diff
@@ -16,3 +16,4 @@ max_request_size=10485760
 unregister_model_timeout=120
 certificate_file=src/test/resources/certs.pem
 management_address=unix\:/tmp/management.sock
+version=0.1.1
```

Comment:
This should come from version.txt

Reply:
addressed this making version a dummy param in test.
note changes in frontend/server/src/test/java/org/pytorch/serve/SnapshotTest.java

Reply:
If we are ignoring this in comparison then can't we remove this. I think that's what we discussed.

Reply:
Done